guides/common/modules/proc_creating-operating-systems.adoc

```diff
@@ -62,7 +68,12 @@ endif::[]
 --family "{client-os-family-hammer}" \
 --major {client-os-major} \
 --media "{client-os-family}" \
+ifndef::debian,ubuntu[]
 --minor {client-os-minor} \
+endif::[]
+ifdef::debian,ubuntu[]
+--release-name "{client-os-codename}" \
+endif::[]
```

Comment:
Positive condition first. Please, swap `ifdef` and `ifndef`.

Reply:
Applied.